### PR TITLE
scheduler: fix ElasaticQuota sync status missing decorate used

### PR DESCRIPTION
**Pull request by jasonliu747**
### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

File: pkg/scheduler/plugins/elasticquota/controller.go [hunk 1]
```diff
@@ -264,6 +266,7 @@ func syncElasticQuotaMetrics(eq *v1alpha1.ElasticQuota, summary *core.QuotaInfoS
 	}
 	RecordElasticQuotaMetric(ElasticQuotaStatusMetric, summary.Used, "used", quotaLabels)
 
+	decorateResource(eq, summary.Used)
```
Comment (eahydra) [inline pkg/scheduler/plugins/elasticquota/controller.go:269]: move before line267

File: pkg/scheduler/plugins/elasticquota/controller.go [hunk 2]
```diff
@@ -176,6 +176,8 @@ func updateElasticQuotaStatusIfChanged(eq *v1alpha1.ElasticQuota, summary *core.
 		}
 	}
 
+	decorateResource(eq, summary.Used)
+
```
Comment (eahydra) [inline pkg/scheduler/plugins/elasticquota/controller.go:180]: remove the space line

File: pkg/scheduler/plugins/elasticquota/controller.go (hunk 1 above)
Comment (jasonliu747) [reply pkg/scheduler/plugins/elasticquota/controller.go]: you are absolutely right!

**Review by eahydra — APPROVED** [review]
/lgtm
/approve

**Pull request merged**